pkg/controller/descheduler/descheduler_controller.go

```diff
@@ -462,8 +478,7 @@ func (r *ReconcileDescheduler) createCronJob(descheduler *deschedulerv1alpha1.De
 							RestartPolicy:     "Never",
 							Containers: []v1.Container{{
 								Name:  "openshift-descheduler",
-								Image: "registry.svc.ci.openshift.org/openshift/origin-v4.0:descheduler", // TODO: Make this configurable too.
```

Comment:
@sjenning - Let me know, if you like this way of exposing image within CR, I see that not many people are doing it this way.